src/lib/pickles/step.ml

```diff
@@ -605,6 +606,15 @@ struct
       let module V = H3.To_vector (E) in
       V.f prev_values_length (M.f prev_with_proofs)
     in
+    assert (
```

Comment:
This is debugging code. Remove, or put behind an `if debug` (with `let debug = false` at the top of the file if it isn't already).

Reply:
ok didn't know that this was debugging code!